mlx/backend/metal/jit_kernels.cpp

```diff
@@ -481,7 +482,7 @@ MTL::ComputePipelineState* get_steel_gemm_masked_kernel(
     kernel_source << metal::utils() << metal::gemm()
                   << metal::steel_gemm_masked()
                   << fmt::format(
-                         steel_gemm_masked_kernels,
+                         fmt::runtime(steel_gemm_masked_kernels),
```

Comment:
This should already be a constexpr 🤔  https://github.com/ml-explore/mlx/blob/main/mlx/backend/metal/jit/steel_gemm.h#L26

Reply:
Looks like you're right. I was patching against an earlier revision. Removed.